wagtail/admin/templates/wagtailadmin/chooser/browse.html

```diff
@@ -5,7 +5,7 @@
     {% trans "Choose a page" as choose_str %}
 {% endif %}
 
-{% include "wagtailadmin/shared/header.html" with title=choose_str subtitle=page_type_names|join:", " search_url="wagtailadmin_choose_page_search" query_parameters="page_type="|add:page_type_string icon="doc-empty-inverse" search_disable_async=True %}
+{% include "wagtailadmin/shared/header.html" with title=choose_str subtitle=page_type_names|join:", " search_url="wagtailadmin_choose_page_search" query_parameters="page_type="|add:page_type_string|add:"&multiple="|add:is_multiple_choice icon="doc-empty-inverse" search_disable_async=True %}
```

Comment:
Note - part of me feels a little bit awkward at just continuing with this logic in the template, happy to refactor this into Python if needed.